src/test/java/com/browserstack/test/suites/login/LoginDataDrivenReadFromCSVTest.java

```diff
@@ -0,0 +1,48 @@
+package com.browserstack.test.suites.login;
+
+import com.browserstack.test.suites.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class LoginDataDrivenReadFromCSVTest extends TestBase {
+
+    @DataProvider(name = "errors")
```

Comment:
name can be login_err_messages or something more contextual?

errors seemed to make me consider if that is some TestNG special value.